### ISSUE TITLE
Error when using Facebook authentication

### ISSUE THREAD

**Issue by stuartbain** [danschultzer/pow_assent#25]
I've just updated my project to use the latest version of pow_assent that removes the Poison and OAuth2 dependencies and am receiving the following error when using the Facebook strategy:

```
[error] #PID<0.1293.0> running MyAppWeb.Endpoint (connection #PID<0.1241.0>, stream id 29) terminated
Server: localhost:4001 (https)
Request: GET /auth/facebook/callback?code=[REMOVED]&state=[REMOVED]
** (exit) an exception was raised:
    ** (ArgumentError) raise/1 and reraise/2 expect a module name, string or exception as the first argument, got: %{body: "{\"error\":{\"message\":\"An active access token must be used to query information about the current user.\",\"type\":\"OAuthException\",\"code\":2500,\"fbtrace_id\":\"Gz3hgqbpPAr\"}}", headers: [{"cache-control", "no-store"}, {"connection", "keep-alive"}, {"date", "Mon, 14 Jan 2019 08:56:28 GMT"}, {"pragma", "no-cache"}, {"www-authenticate", "OAuth \"Facebook Platform\" \"invalid_request\" \"An active access token must be used to query information about the current user.\""}, {"content-length", "167"}, {"content-type", "text/javascript; charset=UTF-8"}, {"expires", "Sat, 01 Jan 2000 00:00:00 GMT"}, {"x-app-usage", "{\"call_count\":0,\"total_cputime\":0,\"total_time\":0}"}, {"facebook-api-version", "v2.12"}, {"x-fb-rev", "4681796"}, {"access-control-allow-origin", "*"}, {"x-fb-trace-id", "Gz3hgqbpPAr"}, {"strict-transport-security", "max-age=15552000; preload"}, {"x-fb-debug", "QeWC2A4PWGHQi+rMc8GXHjZBDVaWW7vKFbZ0YHQe8NYCZK8HO2QJw60j6noSXCr5XAYP1nBYvLwbrxTbus6gWg=="}], status: 400}
        (pow_assent) lib/pow_assent/phoenix/controllers/authorization_controller.ex:117: PowAssent.Phoenix.AuthorizationController.handle_strategy_error/1
        (pow_assent) lib/pow_assent/phoenix/controllers/authorization_controller.ex:1: PowAssent.Phoenix.AuthorizationController.action/2
        (pow_assent) lib/pow_assent/phoenix/controllers/authorization_controller.ex:1: PowAssent.Phoenix.AuthorizationController.phoenix_controller_pipeline/2
        (myapp) lib/myapp_web/endpoint.ex:1: MyAppWeb.Endpoint.instrument/4
        (phoenix) lib/phoenix/router.ex:275: Phoenix.Router.__call__/1
        (myapp) lib/myapp_web/endpoint.ex:1: MyAppWeb.Endpoint.plug_builder_call/2
        (myapp) lib/plug/debugger.ex:122: MyAppWeb.Endpoint."call (overridable 3)"/2
        (myapp) lib/myapp_web/endpoint.ex:1: MyAppWeb.Endpoint.call/2
        (phoenix) lib/phoenix/endpoint/cowboy2_handler.ex:34: Phoenix.Endpoint.Cowboy2Handler.init/2
        (cowboy) /Users/stuart/Development/Elixir/myapp/deps/cowboy/src/cowboy_handler.erl:41: :cowboy_handler.execute/2
        (cowboy) /Users/stuart/Development/Elixir/myapp/deps/cowboy/src/cowboy_stream_h.erl:296: :cowboy_stream_h.execute/3
        (cowboy) /Users/stuart/Development/Elixir/myapp/deps/cowboy/src/cowboy_stream_h.erl:274: :cowboy_stream_h.request_process/3
        (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
```

**Comment by danschultzer** [comment]
Sorry, I forgot to test this properly. It has been fixed now along with #24. I'll test the other strategies, and fix any other bugs and then push a new update. Thanks for letting me know!

**Comment by danschultzer** [comment]
`0.1.0-rc.2` has been released 🚀
